website/docs/r/integration_docker_v2.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: "lacework"
+page_title: "Lacework: lacework_integration_docker_v2"
+description: |-
+  Create and manage Docker V2 container registry integrations
+---
+
+# lacework\_integration\_docker\_v2
+
+Use the Docker V2 Registry integration for private Docker V2 registries only.
+
+~> **Note:** For Docker Hub, ECR, and GCR, use their corresponding container registry types.
+
+The Docker V2 Registry integration functions differently than Lacework's other container registry
+integrations. This integration performs on-demand image assessment via the CLI/API, while the other
+integrations automatically assess images at regular intervals.
+
+Supported Docker V2 registries:
+
+* Azure Container Registry
+* GitLab (On prem 12.8 and cloud)
+* JFrog Artifactory (On prem 7.2.1 and cloud)
+* JFrog Platform (On prem 7.2.1 and cloud)
+
+~> **Note:** You must whitelist the Lacework outbound IPs to allow the vulnerability scanner to communicate with your private registries. See [Lacework Outbound IPs](https://support.lacework.com/hc/en-us/articles/360052140433)
+
+## Set Up Image Assessments
+
+You can, for example, schedule on demand scans as part of a (CI) pipeline as we covered in
+[Up and Running with Lacework and Jenkins](https://www.lacework.com/running-with-jenkins/).
```

Comment:
@scottford-lw I think we should collaborate to have something more meaningful here. Should we Zoom when you get online? 💯